templates/security/osv.html

```diff
@@ -72,7 +72,7 @@ <h2 id="osv-open-format-vuln-management">OSV &mdash; an open format for vulnerab
       <hr class="p-rule" />
       <div class="col">
         <div class="p-section--shallow">
-          <div class="p-image-container is-cover u-hide--small">
+          <div class="p-image-container--2-3 is-cover u-hide--small">
```

Comment:
This one looks a bit off:

- Before

<img width="956" height="1111" alt="image" src="https://github.com/user-attachments/assets/ebb6bf7f-64d0-46b0-b757-566c8b668191" />

- After (zoomed out to capture the change):

<img width="956" height="1111" alt="image" src="https://github.com/user-attachments/assets/0784ce5a-584a-4979-8363-3ba2e4313b39" />